src/main/java/dk/cngroup/lentils/service/convertors/TeamConverter.java

```diff
@@ -0,0 +1,46 @@
+package dk.cngroup.lentils.service.convertors;
+
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.entity.formEntity.TeamDTO;
+import dk.cngroup.lentils.service.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class TeamConverter {
+    private TeamService teamService;
+
+    @Autowired
+    public TeamConverter(final TeamService teamService) {
+        this.teamService = teamService;
+    }
+
+    public TeamDTO fromEntity(final Team team) {
+        final TeamDTO dto = new TeamDTO();
+
+        dto.setTeamId(team.getTeamId());
+        dto.setName(team.getName());
+        dto.setNumOfMembers(team.getNumOfMembers());
+        dto.setPin(team.getPin());
+        dto.setUser(team.getUser());
+
+        return dto;
+    }
+
+    public void toEntity(final TeamDTO dto, final Team team) {
+        team.setTeamId(dto.getTeamId());
+        team.setName(dto.getName());
+        team.setNumOfMembers(dto.getNumOfMembers());
+        team.setPin(dto.getPin());
+        team.setUser(dto.getUser());
+    }
+
+    public List<TeamDTO> getDtosListFromAllTeams() {
+        List<TeamDTO> dtos = new ArrayList<>();
+        teamService.getAll().forEach(team -> dtos.add(fromEntity(team)));
```

Comment:
Toto bych tady uplne nedaval. Ten konverter by mel fungovat tak, ze mu podstrcis entitu a on je prevede na jinou. To, ze tady uz je ten team service trochu svadi k tomu, ze zrovna zavolas find all na ni, ale predstav si, ze budes chtit vytahnout z db teamy podle nejakeho kriteria a pak budes chtit zavolat tento konverter a prevest je. V tom pripade uz nebudes moct pouzit tuto metodu, ale budes muset vytvorit jinou, ktera se bude lisit v tom, ze zavola jinou metodu na repository/service. Takze tady by bylo lepsi kdyby se ta metoda jmenovala `fromEntity(List<Team> teams)` a mol si tam poslat libovoly seznam tymu ;)